Modules/_sqlite/cache.c

```diff
@@ -44,14 +44,27 @@ pysqlite_new_node(PyObject *key, PyObject *data)
     return node;
 }
 
+static int
+node_traverse(pysqlite_Node *self, visitproc visit, void *arg)
+{
+    Py_VISIT(Py_TYPE(self));
```

Comment:
self->key, self->data doesn't be visited in here?

Reply:
> self->key, self->data doesn't be visited in here?

Key is just a string, IIRC (I'll check when I'm back on my computer). Data is a statement object IIRC, so that should be visited. I'll fix it. Thanks!